test/components/idea_controls_test.js

```diff
@@ -172,4 +183,63 @@ describe("<IdeaControls />", () => {
       })
     })
   })
+
+  describe("the vote button", () => {
+    context("when the stage is not idea-generation", () => {
```

Comment:
without looking at the implementation, i'd think that the only time we'd render the vote button is during the voting stage, right? does the category of the idea need to come into play, given that there won't be any action items under _after_ the voting stage is over?